src/network_parameters.cpp

```diff
@@ -78,8 +78,8 @@ static std::map<std::string, std::shared_ptr<nlohmann::json>> registered_network
                 { "default_peers", nlohmann::json::array() }, { "p2pkh_version", 111u }, { "p2sh_version", 196u },
                 { "bech32_prefix", "bcrt" }, { "mainnet", false }, { "liquid", false }, { "development", true },
                 { "csv_buckets", std::vector<uint32_t>{ 144, 4320, 51840 } }, { "bip21_prefix", "bitcoin" },
-                { "electrum_url", "localhost:19002" }, { "spv_enabled", false }, { "tls", false },
-                { "server_type", "green" } })) },
+                { "electrum_url", "localhost:19002" }, { "tls", false }, { "server_type", "green" },
+                { "spv_enabled", false }, { "spv_cross_validation", false }, { "spv_cross_validation_servers", nlohmann::json::array() } })) },
```

Comment:
Please run `./tools/format.sh` to format `src/network_parameters.cpp` correctly.

I recently added https://github.com/Blockstream/gdk/commit/ac593ff6c31948eef99b95068100ae81a81b5499 so that they are run in travis as well

Reply:
I wasn't aware this existed, will do.

Reply:
Formatted in a914e561eae54abb6b7b5e7d853a66cc32df1d48.